Kernel/Process.cpp

```diff
@@ -246,8 +246,8 @@ ErrorOr<NonnullLockRefPtr<Process>> Process::try_create_user_process(LockRefPtr<
     }));
 
     Thread* new_main_thread = nullptr;
-    u32 prev_flags = 0;
```

Comment:
I've called this concept `previous_interrupts_state` before, see https://github.com/SerenityOS/serenity/blob/00db1dffa8fa0da8c06e18e0316ec0726b2d34b0/Kernel/Arch/Processor.h#L15-L22